go/vt/vttablet/tabletserver/health_streamer.go

```diff
@@ -255,3 +297,82 @@ func (hs *healthStreamer) SetUnhealthyThreshold(v time.Duration) {
 		}
 	}
 }
+
+// Reload reloads the schema from the underlying mysql
+func (hs *healthStreamer) Reload() error {
+	hs.mu.Lock()
+	defer hs.mu.Unlock()
+
+	// Schema Reload to happen only on master.
+	if hs.state.Target.TabletType != topodatapb.TabletType_MASTER {
+		return nil
+	}
+
+	ctx := hs.ctx
+	conn, err := hs.conns.Get(ctx)
+	if err != nil {
+		return err
+	}
+	defer conn.Recycle()
+
+	if !hs.initSuccess {
+		hs.initSuccess, err = hs.InitSchemaLocked(conn)
+		if err != nil {
+			return err
+		}
+	}
+
+	qr, err := conn.Exec(ctx, mysql.DetectSchemaChange, 5, false)
```

Comment:
anything significant about the number `5`? Or is it enough that it's nonzero? If the latter, then `1` is a better magic number than `5`.